subjects/rt/audit/README.md

```diff
@@ -0,0 +1,35 @@
+#### Functional
+
+##### Using the ray tracer construct any scene you want, including at least one of all objects. (this can take a while to render, so in the meanwhile you can answer the next questions)
+
+###### Does the image corresponds to the scene you created?
+
```

Comment:
maybe you can add a question that asks the auditor to try and create one of the same scenes given to them (the 4 .ppm images). This way we could make sure that the images where really made using their program. what do you think?

Reply:
it can  be difficult to recreate the same image, but if they already have the code to do it yeah, it is possible

Reply:
ok, so it will be better if we leave it like you already have